lib/index.js

```diff
@@ -50,10 +51,14 @@ AdWords.prototype.loaded = function() {
  * @param {Page} page
  */
 
-AdWords.prototype.page = function() {
+AdWords.prototype.page = function(page) {
+  // https://developers.google.com/adwords-remarketing-tag/asynchronous/
   var remarketing = !!this.options.remarketing;
   var id = this.options.conversionId;
-  var props = {};
+  // it's okay to send custom props even if customers dont use dynamic remarketing
+  // "if this variable is not provided, static remarketing rules can still be generated in AdWords by using URL-based rules."
+  // customers may have custom instance of google params on the window
+  var props = remarketing ? extend(page.properties(), window.google_custom_params) : {};
```

Comment:
If they don't have `remarketing` enabled, we don't want to send anything because Google doesn't do anything with it I believe @f2prateek